src/components/Form/SetAclPermsDocContainerForm.jsx

```diff
@@ -108,21 +122,25 @@ const SetAclPermsDocContainerForm = () => {
 
   return (
     <FormSection
-      title="Permission to Documents Container"
+      title={
+        user === 'personal' ? 'Set Permission to Documents' : 'Request Permissions to Documents'
+      }
       state={state}
       statusType="Permission status"
       defaultMessage="To be set..."
     >
       <Box display="flex" justifyContent="center">
         <form onSubmit={handleAclPermission} autoComplete="off">
           <FormControl>
-            <Typography htmlFor="set-acl-to">Set permissions to username:</Typography>
+            <Typography htmlFor="set-acl-to">
+              {user === 'personal' ? 'Set' : 'Request'} permissions to username:
```

Comment:
Let's remove this for now. We should talk to UX/UI about the best way to send permission requests. Either way, it's not the subject of this PR.

Reply:
I think we could leave the change?